### DISCUSSION TITLE
Init on Android

### DISCUSSION THREAD

**Discussion by RomanoZumbe** [Vinzent03/obsidian-git#647] [Q&A]
How can I initialize my repo on Android?

**Comment by Vinzent03** [comment]
It's documented in the [documentation](https://publish.obsidian.md/git-doc/Getting+Started)

**Reply by RomanoZumbe** [reply]
Thank you! This part was missing in the plugin description in obsidians store and it told me to look at the 'mobile' link (which didn't lead to the repo)
